src/python/WMCore/WMSpec/Steps/Executors/LogArchive.py

```diff
@@ -129,6 +148,11 @@ def execute(self, emulator=None, **overrides):
 
         signal.signal(signal.SIGALRM, alarmHandler)
         signal.alarm(waitTime)
+        try:
+            eosmanager(fileInfo)
+        except:
+            logging.warning("EOS copy failes lfn: %s" % fileInfo['LFN'] )
```

Comment:
Log exception?

Reply:
exception would be better since we can get the traceback. @amaltaro, do you know whether logging writes here?

Reply:
It should be going to the wmagentJob.log file (and unintentionally to _condor_stderr)